tests/test_units.py

```diff
@@ -343,7 +343,7 @@ def setUp(self):
         self.backup = [x for x in HookTest.capitains_units.cts.CTSText_TestUnit.tests]
         self.Epidoc = HookTest.capitains_units.cts.CTSText_TestUnit("/false/path")
         self.Epidoc.scheme = "epidoc"
-        self.frame = """<TEI xmlns="http://www.tei-c.org/ns/1.0">
+        self.frame = r"""<TEI xmlns="http://www.tei-c.org/ns/1.0">
```

Comment:
That's exactly why a raw string is appropriate: The regular expressions in the xml should be passed as part of an xml document, not expanded as a python string first. The data passes through ok, because `\w` isn't a Python string escape, but since Python 3.6 the interpreter warns about it in case they want to add such an escape in the future.

If you had:

    "<p>Use the <code>\n</code> newline escape to insert blank lines.</p>"

Letting the python interpreter replace the `\n` with an actual newline would result in incorrect rendering. It's the same with the xpath regex, unless I'm missing something?

Reply:
Anything else I can do to address your concern here, @PonteIneptique?

Reply:
I'll need to review the issue you talked about but won't have the time before tomorrow (EU Time) I believe

Reply:
Ok, thanks!

Reply:
I am not able to reproduce the warning on the `\w` you are talking about unfortunately...

Reply:
I read https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals
So even if there is no warning, I'll have to follow you on this one. Even though I have to say I am not agreeing with the path taken by python on this one ;)